lib/invisible_captcha.rb

```diff
@@ -36,6 +39,12 @@ def init!
       # If enabled, you should call anywhere in of your layout the following helper, to inject the honeypot styles:
       #  <%= invisible_captcha_styles %>
       self.injectable_styles = false
+      
+      # IP check enabled by default
+      self.ip_enabled = true
+      
+      # A secret key setup in config settings. 'rake secret' will give you a good one.
+      self.secret = Digest::MD5.hexdigest("make_sure_to_set_your_secret_key_in_config_settings_do_not_use_default")
```

Comment:
Maybe be can default to something like Devise does:

https://github.com/heartcombo/devise/blob/45b831c4ea5a35914037bd27fe88b76d7b3683a4/lib/generators/templates/devise.rb#L17

Reply:
Accessing the key is different for each rails version, so it was easier and cleaner to have the user set one.

Reply:
I added the code Devise used for secret key finder, but the way the gem loads, Rails.application does not load before setting the value. @markets can take a look as to the best way to get the loading order correct. This is the first gem I have contributed to, so not sure of the best structure to do this.

Reply:
but the line I copied above is just:

```
config.secret_key = '<%= SecureRandom.hex(64) %>'
```

nothing related to Rails, but IMO better than `Digest::MD5.hexdigest("make_sure_to_set_your_secret_key_in_config_settings_do_not_use_default")` which requires the user to change it.